backend/capellacollab/projects/routes.py

```diff
@@ -85,15 +87,19 @@ def update_project(
     ),
     db: orm.Session = fastapi.Depends(database.get_db),
 ) -> models.DatabaseProject:
-    new_slug = slugify.slugify(patch_project.name)
-    if crud.get_project_by_slug(db, new_slug) and project.slug != new_slug:
-        raise fastapi.HTTPException(
-            status_code=status.HTTP_409_CONFLICT,
-            details={
-                "reason": "A project with a similar name already exists.",
-                "technical": "Slug already used",
-            },
-        )
+    if patch_project.name:
+        new_slug = slugify.slugify(patch_project.name)
```

Comment:
The new slug is never updated in the database.

Reply:
Isn't it? This here is just the safety check/ validation that there is no other project with the same slug. At the end of the function/route, then the `update_project` function is called which makes the actual update. In theory, we could also set `project.slug = new_slug` in here but IMO that should be in the responsibility of the actual update function.